src/registrar/templates/application_org_election.html

```diff
@@ -6,7 +6,11 @@ <h2 class="margin-bottom-05">
     Is your organization an election office?  <abbr class="usa-hint usa-hint--required" title="required">*</abbr>
   </h2>
 
-  <p>Answer “yes” if the primary purpose of your organization is to manage elections.</p>
+  <p>An election office is a government entity whose <i>primary</i> responsibility is overseeing elections and/or conducting voter registration.
+  <br>
```

Comment:
*comment (non-blocking)*: I think that a more semantic approach would be to use two paragraphs `</p><p>` for keeping these two statements separate rather than the purely visual markup with `<br/>` tags.